contracts/scripts/measure_gas/token.ts

```diff
@@ -0,0 +1,68 @@
+import { BigNumber, Signer } from "ethers";
+import {
+  ActionData,
+  BlsWalletContracts,
+  BlsWalletWrapper,
+} from "../../clients/src";
+import { MockERC20 } from "../../typechain-types";
+import { processBundles } from "./transaction";
+import { GasMeasurementContext } from "./types";
+
+export const getTransferAmount = (ctx: GasMeasurementContext): BigNumber => {
+  return BigNumber.from(ctx.rng.int(1, ctx.blsWallets.length));
+};
+
+export const createTransferAction = (
+  ctx: GasMeasurementContext,
+  to: string,
+): ActionData => ({
+  contractAddress: ctx.contracts.testToken.address,
+  encodedFunction: ctx.contracts.testToken.interface.encodeFunctionData(
+    "transfer",
+    [to, getTransferAmount(ctx)],
+  ),
+  ethValue: 0,
```

Comment:
This does not include a `tx.origin` payment, which is essential.

Makes sense to split it out though: https://github.com/web3well/bls-wallet/issues/508.

Reply:
Since these measurements go directly through the VG contract, I don't think it's relevant at this level as the VG doesn't care about `tx.origin`. If we decided in the future to do a more full stack measurement using an agg instance, we can add in that script.

Reply:
I'm not sure I understand your point.

I can see how ops that don't pay `tx.origin` are useful for internal comparisons (e.g. is the gas usage going up or down), but I don't think that's a relatively low priority.

The priority is whether we're beating non-bls transactions, and by how much. For this purpose, ops that don't compensate `tx.origin` are not relevant.

Reply:
(Let's continue this outside of the current PR, since it's non-blocking.)